Dockerfile

```diff
@@ -28,3 +28,6 @@ COPY --from=client /project/static/vue /project/static/vue
 COPY --from=client /project/webpack-stats.json /project/webpack-stats.json
 
 EXPOSE 80 443
+
+HEALTHCHECK --interval=1m --timeout=3s \
```

Comment:
Port 8890 is the one mapped by docker-compose -- the service itself I think exposes port 8888. Does this still work due to some quirk of how HEALTHCHECK works? Maybe it should be in the docker-compose.yml instead. Which reminds me, does this have any effect on production if it's in the Dockerfile?

*Edit*: Reading more about it, it seems like it should be checking port 8888, but that it's reasonable to keep it in the Dockerfile.

Reply:
I wasn't sure here. We `EXPOSE` 80 and 443, but compose is running a custom "command", and mapping 8890->8888?

Reply:
moved to compose file. this really needs to take the entrypoint / cmd into account because of how we use the container.